conans/test/model/other_settings_test.py

```diff
@@ -93,7 +93,7 @@ def invalid_settings_test2(self):
 class SayConan(ConanFile):
     name = "Say"
     version = "0.1"
-    settings = {"os": ["Windows", "Linux", "Macos"], "compiler": ["Visual Studio"]}
+    settings = {"os": ["Windows", "Linux", "Macos", "FreeBSD"], "compiler": ["Visual Studio"]}
```

Comment:
I think this test change might not be necessary, it is just an example of some settings and error messages, but it is not related to the real settings actually used.